py4DSTEM/process/diskdetection/diskdetection_parallel.py

```diff
@@ -1,11 +1,11 @@
 # stdlib
 import os
-import pickle
 import tempfile
 from time import time
 
 # 3rd party
 import numpy as np
+import dill
```

Comment:
Just confirming - this should not be a new dependency, `dill` is included in base python past 3.whatever, right?